src/helpers/send-bulk-message-helper.js

```diff
@@ -0,0 +1,66 @@
+export const isPotentialMobileNumber = (number) => {
+  if(!number){
+    return false
+  }
+  let trimmedNumber = number.trim();
+  if (
+    (trimmedNumber?.substring(0, 2) == "07") && trimmedNumber.length == 11||
+    (trimmedNumber?.substring(0, 1) == "7") && trimmedNumber.length == 10 ||
+    (trimmedNumber?.substring(0, 4) == "+447") && trimmedNumber.length == 13||
+    (trimmedNumber?.substring(0, 3) == "447") && trimmedNumber.length == 12)
+    {
+    return true
+    }
+  return false
+}
+export const containsPotentialMobileNumer = (number) => {
+  if(!number){
+    return false
+  }
+  let contansPotentialMobile=[]
+  if(isPotentialMobileNumber(number)){
+    number
+    return true
+  } 
+
+  if(number.includes('/')){
+    let numbers = number.split("/")
+    numbers.forEach(number => {
+      if(isPotentialMobileNumber(number)){
+        contansPotentialMobile.push(true)
+      }
+    });
+  }
+  if(contansPotentialMobile.includes(true)){
+    return true
+  }
+  return false
+}
+
+export const appendMobileContact = (mobileContacts, number, callback) =>{
+  if(number && number.includes('/')){
+    let numbers = number.split("/")
+    numbers.forEach(number => {
+      if(number && isPotentialMobileNumber(number)){
```

Comment:
`if(isPotentialMobileNumber(number))` should be enough cause you check if number exists in the method itself